src/components/task-form/task-form.component.js

```diff
@@ -0,0 +1,60 @@
+import { useRef, useState } from "react";
+import GetPriorityOptions from "../../services/get-priority-options";
+
+const TaskForm = (props) => {
+  const { addNewTask, closeFormHandler } = props;
+  const [task, setTask] = useState({
+    name: "",
+    dueDate: "",
+    priority: "low",
+  });
+
+  const buttonEl = useRef();
```

Comment:
there's no much use for this `buttonEl` ref. I would suggest removing it

Reply:
6b51abaa16aeaa0f38c400065855bc66d39d70cb Deleted as requested since this was not being used.